gobbler/views/welcome.html

```diff
@@ -0,0 +1,7 @@
+<template name="welcome">
+  {{> navBar }}
+  <div class="welcome">
+    <p>Welcome to Gobbler!</p>
+    <p>Remove the nav bar from this page after we build login event redirect.</p>
```

Comment:
Just an FYI, the login redirect is currently commented out in the server.js file. I was having an issue where it was redirecting at random points. I didn't spend too long working on it though.